### ISSUE TITLE
Can't install on Deepin due to missing GLIBC_2.29

### ISSUE THREAD

**Issue by vasilvestre** [oven-sh/bun#1195]
### Version

latest

### Platform

Linux vsilvestre-PC 5.10.101-amd64-desktop #20.00.05.06-release SMP Mon Mar 14 09:34:55 CST 2022 x86_64 GNU/Linux

### What steps will reproduce the bug?

I installed it via your command

### How often does it reproduce? Is there a required condition?

All the time

### What is the expected behavior?

Having a functionnal bun install

### What do you see instead?

I don't really know how to fix this, I'm still searching a workaround

**Comment by LoiLock** [comment]
Install glibc?
```bash
wget -c https://ftp.gnu.org/gnu/glibc/glibc-2.29.tar.gz
tar -zxvf glibc-2.29.tar.gz
cd glibc-2.29
./configure --prefix=/opt/glibc
make 
make install
```

**Comment by confact** [comment]
Have the same issue as @vasilvestre - I tried to add the bullseye (Debian) repository to apt to get the latest libc but of course, it didn't work.

Right now, I am stuck at:
```
libc-bin is already the newest version (2.28.19-1+dde).
libc6 is already the newest version (2.28.19-1+dde).
```

I think this will be solved when Deepin 23 is released, it is currently in preview. 

I would love a temporary fix until then.
@LoiLock your example seems to not work.

**Comment by confact** [comment]
Answer to myself and @vasilvestre:

Using docker as a temporary fix for now. It seems to work ok.

**Comment by vasilvestre** [comment]
> Install glibc?
> 
> ```shell
> wget -c https://ftp.gnu.org/gnu/glibc/glibc-2.29.tar.gz
> tar -zxvf glibc-2.29.tar.gz
> cd glibc-2.29
> ./configure --prefix=/opt/glibc
> make 
> make install
> ```

Same as @confact , I have the same error despite the install

**Comment by Electroid** [comment]
This has now been fixed in [Bun 0.3.0](https://bun.sh/blog/bun-v0.3.0) 🎉
